### DISCUSSION TITLE
How I can add keyboard layout?

### DISCUSSION THREAD

**Discussion by Bogdan1232-ad** [prasanthrangan/hyprdots#2086] [Q&A]
I'm beginner of linux and I don't understand why 
localectl set-x11-keymap --no-convert us,ru pc105 "" grp:alt_shift_toggle
doesn't work. My model of keyboard is pc105. Help me please
I did locale-gen.

**Comment by kRHYME7** [comment]
read Hyprland wiki

**Reply by Bogdan1232-ad** [reply]
I've done all from arch wiki(I've installed arch hyprland)

**Reply by kRHYME7** [reply]
> I've done all from arch wiki(I've installed arch hyprland)


so how did you add the keyboard layout in your hyprland configuration?
